ark/type/keywords/string/base64.ts

```diff
@@ -0,0 +1,20 @@
+import type { Predicate, of } from "../inference.ts"
+import { regexStringNode } from "./utils.ts"
+
+declare namespace string {
+	export type base64 = of<string, Predicate<"base64">>
+	export type base64url = of<string, Predicate<"base64url">>
+}
+
+// https://stackoverflow.com/a/475217/1255873
+export const base64 = regexStringNode(
+	/^(?:[A-Za-z0-9+/]{4})*(?:[A-Za-z0-9+/]{2}==|[A-Za-z0-9+/]{3}=)?$/,
+	"alphanumeric characters and +/ with the correct amount of = padding"
```

Comment:
I would just describe this as "base64-encoded" or similar so the error for a bad string would be something like `"V29yZA"` would be `must be base64-encoded (was "V29yZA")`.

Generally people using this type should know what format to expect, but if you're concerned about opacity, you could describe it as `base64-encoded (see https://link.to/spec)`